### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1] Role = Tree should not contain multiple UI elements.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11279]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/breadcrumb
Below code found by Access assistant tool for breadcrumb
This FD-BREADCRUMB with a role attribute value of 'tree' should not contain input elements (without an assigned ARIA role), button elements (without an assigned ARIA role), text area elements (without an assigned ARIA role), select elements (without an assigned ARIA role) or A elements (without an assigned ARIA role).

Code Snippet: <div class="fd-breadcrumb fd-dynamic-page__breadcrumb" role="tree" _ngcontent-ng-c2538891962=""><div style="" class="fd-overflow-layout" showmoreposition="left"><div style="display: none;" class="fd-overflow-layout__more"><div class="is-cozy"></div></div><div class="fd-overflow-layout__items-container"><div class="fd-overflow-layout__items-wrapper"><div class="fd-overflow-layout__items"><div class="fd-overflow-layout__item fd-overflow-layout__item--first" fdoverflowlayoutitemcontainer=""><div fdoverflowlayoutitem=""><div class="fd-breadcrumb__item" _ngcontent-ng-c2538891962=""><a class="fd-link" _ngcontent-ng-c2538891962=""><span class="fd-link__content" _ngcontent-ng-c2538891962="">Shared Keys</span></a></div></div></div><div class="fd-overflow-layout__item" fdoverflowlayoutitemcontainer=""><div fdoverflowlayoutitem=""><div class="fd-breadcrumb__item" _ngcontent-ng-c2538891962=""><a class="fd-link" _ngcontent-ng-c2538891962=""><span class="fd-link__content" _ngcontent-ng-c2538891962="">adb9ddf1-602f-45a8-aeeb-8914b88a801d</span></a></div></div></div><div class="fd-overflow-layout__item fd-overflow-layout__item--last" fdoverflowlayoutitemcontainer=""><div fdoverflowlayoutitem=""><div class="fd-breadcrumb__item" _ngcontent-ng-c2538891962=""></div></div></div></div></div></div></div></div>

One way to avoid inappropriate nesting of elements is to ensure that page source validates to a formal grammar by running it through an HTML validator. Developers should remove nested elements that should not be nested. This may take the form of removing one of the two elements and ensuring the remaining element provides the proper textual equivalent. If the two active elements perform different actions, the elements should be 'adjacent' to each other in the markup, rather than being nested, and the specific action of each element should be identified. Similar issues may occur when links and other content are embedded in form labels.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
•	Download Access assistant tool from level access.
•	For registered AMP user, login with your SAP email ID into SAP AMP instance: sap.levelaccess.net
Use any breadcrumb sample code
This FD-BREADCRUMB with a role attribute value of 'tree' will contain input elements (without an assigned ARIA role), button elements (without an assigned ARIA role), text area elements (without an assigned ARIA role), select elements (without an assigned ARIA role) or A elements (without an assigned ARIA role).

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?
